unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BlockerController/WorldBlockersController.cs

```diff
@@ -36,35 +36,43 @@ public class WorldBlockersController
             new Vector2Int(-1, 1)
         };
 
-        public WorldBlockersController(ISceneHandler sceneHandler, IBlockerInstanceHandler blockerInstanceHandler, DCLCharacterPosition characterPosition)
+        public void Initialize(ISceneHandler sceneHandler, IBlockerInstanceHandler blockerInstanceHandler, DCLCharacterPosition characterPosition)
```

Comment:
For now, `WorldBlockersController` is initialized by this method instead of the constructor. This is done because we can't replace the class instance in the `Environment` initialization stage because its read only. 

Not a fan of having instantiation and initialization split like this, but fixing that issue would be a pretty big change. This should work for now.